packages/ubuntu_bootstrap/lib/pages/install/install_page.dart

```diff
@@ -6,22 +7,31 @@ import 'package:ubuntu_bootstrap/l10n.dart';
 import 'package:ubuntu_bootstrap/pages/install/bottom_bar.dart';
 import 'package:ubuntu_bootstrap/pages/install/install_model.dart';
 import 'package:ubuntu_bootstrap/pages/install/slide_view.dart';
+import 'package:ubuntu_bootstrap/services.dart';
 import 'package:ubuntu_provision/ubuntu_provision.dart';
 import 'package:ubuntu_widgets/ubuntu_widgets.dart';
 import 'package:ubuntu_wizard/ubuntu_wizard.dart';
 import 'package:yaru_icons/yaru_icons.dart';
 import 'package:yaru_widgets/yaru_widgets.dart';
 
 class InstallPage extends ConsumerWidget with ProvisioningPage {
-  const InstallPage({super.key});
+  InstallPage({super.key});
+
+  bool liveSession = true;
```

Comment:
This should be stored in the `installModelProvider`, since you shouldn't have mutable state here.

Reply:
Ok, I think that it is done... I still have to test it, anyway.

Reply:
Looks correct now, good job!